renderer/reducers/info.js

```diff
@@ -186,29 +187,21 @@ export const receiveInfo = data => async (dispatch, getState) => {
 // ------------------------------------
 
 const ACTION_HANDLERS = {
-  [SET_HAS_SYNCED]: (state, { hasSynced }) => ({
-    ...state,
-    hasSynced,
-  }),
-  [GET_INFO]: state => ({ ...state, infoLoading: true }),
+  [SET_HAS_SYNCED]: (state, { hasSynced }) => {
+    state.hasSynced = hasSynced
+  },
+  [GET_INFO]: state => {
+    state.infoLoading = true
+  },
   [RECEIVE_INFO]: (state, { data }) => {
-    const chain = get(data, 'chains[0].chain')
-    const network = get(data, 'chains[0].network')
-    return {
-      ...state,
-      infoLoading: false,
-      infoLoaded: true,
-      chain,
-      network,
-      data,
-    }
+    state.infoLoading = false
+    state.infoLoaded = true
+    state.data = data
+    state.chain = get(data, 'chains[0].chain')
+    state.network = get(data, 'chains[0].network')
   },
 }
 
-// ------------------------------------
```

Comment:
This section heading should remain for consistency

Reply:
hm. i have removed those in #2513 too, should I reinstate them too?

Reply:
You removed the one above the reducer declaration:
```
// ------------------------------------
// Reducer
// ------------------------------------
```
Which I think is fine since we are no longer declaring a reducer but simply exporting the reducer as defined by createReducer.

But I think we should keep the heading above the selectors, which is what I was asking you to bring back here:
```
// ------------------------------------
// Selectors
// ------------------------------------
```

Reply:
ok got it

Reply:
updated